consensus/log/messages.go

```diff
@@ -49,6 +49,6 @@ func (m *Messages) voteSet(voteType vote.Type) *voteset.VoteSet {
 		return m.changeProposerVotes
 	}
 
-	logger.Panic("unexpected vote type %d", voteType)
```

Comment:
Is it safe panic here?

Reply:
We checked the vote_type parameter in the `SanityCheck()` function. 
Returning an error at this point would require multiple error checks in various other locations.